src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -1007,7 +1007,7 @@ class EventView {
     return defaultOption;
   }
 
-  getDisplayOptions() {
+  getDisplayOptions(): SelectValue<string>[] {
```

Comment:
Just noticed this, but `DISPLAY_MODE_OPTIONS` is of type `SelectValue<string>[]` but `getDisplayOptions` can optionally add `disabled` then later its treated like a type `Option` from https://github.com/getsentry/sentry/blob/c8a6cc853768f150ace339ead2abc64657138931/src/sentry/static/sentry/app/components/charts/optionSelector.tsx#L12

I'll probably change the type definition of `SelectValue<T>` back and update all of these to use `Option` instead.

Reply:
Actually, I think it makes more sense to replace `Option` with `SelectValue<string>` and keep the `disabled?` on `SelectValue<T>`.

Reply:
Do other consumers of `SelectValue` also need/use the disabled key?